community-membership.md

```diff
@@ -50,8 +50,8 @@ remain active contributors to the community.
 - Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
     - Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating
       on issues, etc.
-    - Sponsors must be reviewers or approvers in at least 1 OWNERS file (in any repo in the Kubernetes GitHub
-      organization)
+    - Sponsors must be reviewers or approvers in at least 1 OWNERS file either in any repo in the [Kubernetes org],
+    or the org they are sponsoring for.
```

Comment:
I like the addition, but we may want to expand on it with a small example along the lines of:
 _"Example: A sponsor who is only an reviewer or approver within a Kubernetes-sigs project, may only sponsor for the Kubernetes-sigs org"_

It's sort of expanded upon further down in the Kubernetes ecosystem part:
_"However, if you are a member of any of the related Kubernetes GitHub organizations but not of the Kubernetes org, you will need explicit sponsorship for your membership request."_

but I think that part is being skimmed over :x